src/pages/iou/request/step/IOURequestStepParticipants.tsx

```diff
@@ -95,8 +94,6 @@ function IOURequestStepParticipants({
 
     // We need to set selectedReportID if user has navigated back from confirmation page and navigates to confirmation page with already selected participant
     const selectedReportID = useRef<string>(participants?.length === 1 ? (participants.at(0)?.reportID ?? reportID) : reportID);
-    // We can assume that shouldAutoReport is true as the initial value is not used. shouldAutoReport is only used after the selectedReportID changes in addParticipant where we'd update shouldAutoReport too
-    const shouldAutoReport = useRef(true);
```

Comment:
This logic was needed back then because there was some overlapping with the selected workspace (it meant both a workspace and also selfDM - in a sense). That's now sorted out.